config/crd/bases/operator.kyma-project.io_watchers.yaml

```diff
@@ -340,6 +340,11 @@ spec:
                   type: string
                 description: LabelsToWatch describes the labels that should be watched
                 type: object
+              manager:
+                description: |-
+                  Manager specifies the component responsible for handling webhook requests.
+                  If not set, falls back to the 'operator.kyma-project.io/managed-by' label.
```

Comment:
The description here is wrong. The `manager` field does not fall back to anything.